content/en/users/tutorials/data-transfer-grid-storage/_index.md

```diff
@@ -10,7 +10,12 @@ description: >
 ## Overview
 
 This tutorial describes the EGI Data Transfer using FTS transfers services and
-WebFTS.
+WebFTS. In the following paragraps you will learn how to:
+
+- use the [FTS](#fts-client-usage) command line client
```

Comment:
### 
```suggestion
- use the [FTS](#fts-client-usage) command-line ### client
```

Reply:
@andrea-manzi  I'm a bit doubtful about this change, the `###` are quite weird.

Reply:
Fixed it.